### DISCUSSION TITLE
Add the ability for the server adition to be displayed in server version in server properties

### DISCUSSION THREAD

**Discussion by computertech1012** [BearWare/TeamTalk5#2144] [Ideas]
Hi,
I think it should be possible for the server aiditon, such as TeamTalk Pro Server or TeamTalk Standard server to be known to the clients so that the users can know what version or adition of the server is being ran or that they are connected to.
For example, Server Version, followed by S for standard or P for pro would be a good idea.
@bear101 What do you think of this idea?
Thanks for reading!

**Comment by bear101** [comment]
Does it matter if the remote server is TeamTalk Standard or TeamTalk Pro? If the server is encrypted or using BearWare.dk WebLogin then you know it's a TeamTalk Pro server.

**Reply by computertech1012** [reply]
Hi,
@bear101 I guess not, but there are some servers, for example, like the US server the first server in the list, wihch is not encrypted. But it is a pro server because we can access bearware login. So it would be nice for the client to know if it's a pro server or not because maybe there is also someone running a pro server that doesn't have bearware web login enabled or something. Just a thought.
Thanks for reading!
